custom/languages/Dutch/admin.php

```diff
@@ -16,7 +16,7 @@
 	 *  Admin controle paneel
 	 */
 	// Login
-	're-authenticate' => 'Gelieve opnieuw te verifiëren',
+	're-authenticate' => 'Gelieve opnieuw te verifiÃ«ren',
```

Comment:
@Headhunterzpw Thanks for the changes, I was just wondering if `verifiÃ«ren` is intentional on this line?